HOWTO/LICENSE-HEADERS.md

```diff
@@ -134,6 +134,18 @@ the format described by the [REUSE Copyright Spec](https://reuse.software/spec-3
 > will check that this format is followed and will automatically update the copyright
 > year when any changes are made.
 
+Below, we explictly state the formats we allow and their use cases (we allow a subset of [REUSE Copyright Spec](https://reuse.software/spec-3.3/#format-of-copyright-notices)):
```

Comment:
> The copyright notice must start with either `Copyright ` or `SPDX-FileCopyrightText: `
> followed by the holders of the copyright. It is highly recommended that you follow
> the format described by the [REUSE Copyright Spec](https://reuse.software/spec-3.3/#format-of-copyright-notices).

I think that this text should be replaced with what you have written below, as we now require a certain format on the copyright notice.